.buildkite/pipeline.yml

```diff
@@ -98,6 +98,29 @@ steps:
                     - BUILDKITE_PULL_REQUEST_BASE_BRANCH=${BUILDKITE_PULL_REQUEST_BASE_BRANCH}
                     - GITHUB_PR_LABELS=${GITHUB_PR_LABELS}
 
+  - label: "Trigger Xpack/Winlogbeat"
```

Comment:
Please update your branch with the latest changes to this file from https://github.com/elastic/beats/pull/38870

The main differences are:
- the diff command
- env vars which need to be propagated to the triggering job

Reply:
Done